crates/turbopack-ecmascript/tests/tree-shaker/analyzer/shared-and-side-effects/output.md

```diff
@@ -0,0 +1,416 @@
+# Items
+
+Count: 11
+
+## Item 1: Stmt 0, `Normal`
+
+```js
+console.log("Hello");
+
+```
+
+- Side effects
+- Reads: `console`
+
+## Item 2: Stmt 1, `VarDeclarator(0)`
+
+```js
+const value = externalFunction();
+
+```
+
+- Declares: `value`
+- Reads: `externalFunction`
+- Write: `value`
+
+## Item 3: Stmt 2, `VarDeclarator(0)`
+
+```js
+const value2 = externalObject.propertyWithGetter;
+
+```
+
+- Declares: `value2`
+- Reads: `externalObject`
+- Write: `value2`
```

Comment:
This should be flagged as side effect as it accesses a property on an object which might have an getter.